src/BlockTypes/ClassicTemplate.php

```diff
@@ -80,14 +109,39 @@ protected function render( $attributes, $content, $block ) {
 			);
 
 			return $this->render_archive_product();
-		} else {
-			ob_start();
+		}
+
+		ob_start();
 
-			echo "You're using the ClassicTemplate block";
+		echo "You're using the ClassicTemplate block";
```

Comment:
Is this debugging or expected?

Reply:
>Only the indent was fixed. This already existed. It seems to be some fallback but judging by L67 I think it's ok returning nothing and get rid of the ob_ methods

@gigitux once again, do you agree?

Reply:
I don't have a strong opinion. Having a fallback value could help us to understand that there is an issue with some conditions above. Instead, with a blank canvas, we could think that it is a Gutenberg issue.